utils/augmentation.py

```diff
@@ -60,6 +60,13 @@ def compose_transforms(params, dataset, type='', ignore_index=None):
                 lst_trans.append(RandomCrop(sample_size=crop_size, ignore_index=ignore_index))
 
     if type == 'totensor':
+        if not dataset == 'trn' and random_radiom_trim_range:  # Contrast stretching at eval. Use mean of provided range
+            RadiometricTrim.input_checker(random_radiom_trim_range)  # Assert range is number or 2 element sequence
+            if isinstance(random_radiom_trim_range, numbers.Number):
+                trim_at_eval = random_radiom_trim_range
+            else:
+                trim_at_eval = round((random_radiom_trim_range[-1] - random_radiom_trim_range[0]) / 2, 1)
+            lst_trans.append(RadiometricTrim(random_range=[trim_at_eval, trim_at_eval]))
         if scale:
             lst_trans.append(Scale(scale))  # TODO: assert coherence with below normalization
```

Comment:
We need to sort a uniform scaling function, so as not to introduce bugs while using radiometric trim.

Reply:
what do you mean? Radiometric Trim augmentation should not have any impact on scaling: the output values of RadiometricTrim are in the same range as the input values (ex.: 0 to 255 for uint8), so scaling function doen's need to "know" if radiomtric trim is being performed.

Reply:
okay, sounds good.